map_data/state_regions/07_south_america.txt

```diff
@@ -878,10 +878,18 @@ STATE_COTE_DU_CAFE = {
     capped_resources = {
         bg_logging = 11
         bg_fishing = 9
+        bg_iron_mining = 40
```

Comment:
Any reason for this much iron and lead resources?
Does this have any balance reason, flavor or historical?